src/test/integration_tests/internal/test_encryption.js

```diff
@@ -403,6 +403,7 @@ mocha.describe('Encryption tests', function() {
             };
             compare_secrets(secrets, system_store_account.master_key_id._id);
         });
+        // TODO: Add test for multiple secret scenario test cases
```

Comment:
I'm not sure you need to add this comment here.
Once someone tests it will also edit whatever files are needed.
(fewer changed files in the PR).
You can note it as a GAP.